### PR TITLE
Add Red Hat policy configuration documentation

### PR DESCRIPTION
**Pull request by zregvart**
Sourced from https://github.com/release-engineering/rhtap-ec-policy/

### CODE REVIEW

**Comment by zregvart** [comment]
Requires https://github.com/release-engineering/rhtap-ec-policy/pull/3

File: website/config/_default/menu.toml [hunk 1]
```diff
@@ -53,6 +53,12 @@
     url = '/docs/user-guide/main'
     weight = 70
 
+  [[main]]
+    parent = 'Documentation'
+    name = 'Red Hat Policy configuration'
```
Comment (simonbaird) [inline website/config/_default/menu.toml]: Maybe?
```suggestion
    name = 'Red Hat Policy Data'
```

Comment (simonbaird) [reply website/config/_default/menu.toml]: Then line 46 could be this maybe:

```diff
-    name = 'Policies'
+    name = 'Policy Rules'
```

Comment (simonbaird) [reply website/config/_default/menu.toml]: (An idea for consideration, not a blocker.)

**Review by simonbaird — APPROVED** [review]

**Comment by zregvart** [comment]
There is a version of the npm package (extension from [release-engineering/](https://github.com/release-engineering/rhtap-ec-policy) that I pushed manually, so the build succeeds now that the rest of the Antora bits have been merged there as well. I guess this is now okay to merge. The thing to sort out is the automatic publishing from https://github.com/release-engineering/rhtap-ec-policy of the npm package, but that is not a concern here.

**Pull request merged**